infoschema/infoschema.go

```diff
@@ -46,6 +46,8 @@ var (
 	ErrTableExists = terror.ClassSchema.New(codeTableExists, "Table '%s' already exists")
 	// ErrTableDropExists returns for dropping a non-existent table.
 	ErrTableDropExists = terror.ClassSchema.New(codeBadTable, "Unknown table '%s'")
+	// ErrUserDropExists returns for dropping a non-existent user.
+	ErrUserDropExists = terror.ClassSchema.New(codeBadUser, "User %s does not exist.")
```

Comment:
I think it is better to define the error message in the parser repo.

Reply:
It seems the same kind error message all defined here, no any in parser repo, do you means put all this Err* in parser repo?

Reply:
Hah, maybe we can use another PR to move all the error message in this file to the parser repo?